packages/app/src/setupProxy.js

```diff
@@ -4,7 +4,9 @@ module.exports = function(app) {
     app.use(
       '/api',
       createProxyMiddleware({
-        target: process.env.REACT_APP_API_URL,
+        // target: process.env.REACT_APP_API_URL,
+        target: 'http://localhost:3000',
```

Comment:
This would break the proxy in deployed environments. I assume this just slipped in by mistake :)

Reply:
Whoops! Yeah, that wasn't supposed to be in there. I updated the branch to include all these changes. Thank you!